features/rule_language.feature

```diff
@@ -133,4 +135,25 @@ Feature: rule_language
     And It should log "<script>:8:in `test'" within 5 seconds
     And It should log "<script>:17:in `block in <main>'" within 5 seconds
     And It should log "<script>:15:in `<main>'" within 5 seconds
-    And It should not log 'This one works!' within 10 seconds
+    But It should not log 'This one works!' within 10 seconds
+
+  @log_level_changed
+  Scenario: Native java exceptions are handled
+    Given log level INFO
+    And code in a rules file
+      """
+      def test
+        Java::JavaLang::Integer.parseInt('k')
+      end
+
+      rule 'test' do
+        on_start
+        run { test }
```

Comment:
Just curious, did this not work in a run block?  Why call out to the method?

Reply:
Just to make sure the method calls are included in the stack trace.

Reply:
makes sense.